### ISSUE TITLE
How to set it up

### ISSUE THREAD

**Issue by tylrhas** [xenotropic/timecat#1]
I would like for you to help me set up this application. Thanks a ton!

**Comment by xenotropic** [comment]
Ok I'll work on some setup documentation, could be a couple of weeks though.

**Comment by tylrhas** [comment]
Thank you!

**Comment by tylrhas** [comment]
Also any chance i can see a live demo?

**Comment by xenotropic** [comment]
Yes, send an email to joe at xenotropic dot net just so I have the email address to set it up with.
